library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
-        ** Medication-withdrawal-or-reduction-during-recording
+        ** Medication-withdrawal-or-reduction-during-recording <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
     * Eye-modulator
-        ** Manual-eye-closure
+        ** Manual-eye-closure <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** Manual-eye-opening
+        ** Manual-eye-opening <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
     * Stimulation-modulator
-        ** Intermittent-photic-stimulation {suggestedTag=Intermittent-photic-stimulation-effect}
+        ** Intermittent-photic-stimulation <nowiki> {suggestedTag=Intermittent-photic-stimulation-effect} [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue,valueClass=numericClass, unitClass=frequencyUnits} </nowiki>
-        ** Auditory-stimulation
+        ** Auditory-stimulation <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-        ** Nociceptive-stimulation
+        ** Nociceptive-stimulation <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
             *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
-    * Hyperventilation
-        ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Physical-effort
+    * Hyperventilation <nowiki> [When selecting hyperventilation from the list, the user is prompted to score the quality of the hyperventilation (excellent effort, good effort, poor effort, refused the procedure, unable to do the procedure). (Source: Beniczky ea 2017, Section 4, Table 2.)]</nowiki>
+        ** Hyperventilation-refused-procedure
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Hyperventilation-poor-effort
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Hyperventilation-good-effort
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+        ** Hyperventilation-excellent-effort
+            *** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
+
+    * Physical-effort <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Cognitive-task
+    * Cognitive-task <nowiki> [Source: Beniczky ea 2017, Table 2.]</nowiki>
         ** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-    * Other-modulator-or-procedure {requireChild}
+    * Other-modulator-or-procedure <nowiki> {requireChild} [Free text describing other modulators or procedures, Source: Beniczky ea 2017, Table 2.]</nowiki>
```

Comment:
Probably the Source phrase should be in parentheses, like other instances:
[Free text describing other modulators or procedures (Source: Beniczky ea 2017, Table 2.)]

Reply:
fixed, thanks!